react-components/src/lib.js

```diff
@@ -32,7 +32,9 @@ const getDefaultTheme = () => createMuiTheme({
     typography: {
         button: {
             textTransform: "none",
-        }
+        },
+        useNextVariants: true,
```

Comment:
I'm wondering if we should hold off on these updates.

By just setting `useNextVariants: true`, the delete icons in the `EditMetadataTemplate` expansion panel headers go from looking like this
![screen shot 2019-02-27 at 12 54 10 pm](https://user-images.githubusercontent.com/996408/53845701-50b91900-3f67-11e9-8b75-830b20a8e78a.png)
to looking like this
![screen shot 2019-03-05 at 4 52 06 pm](https://user-images.githubusercontent.com/996408/53845708-56aefa00-3f67-11e9-8069-6d6a404ab65c.png)
This happens in both Chrome and Firefox, and it doesn't matter which `Typography` variants are used in `EditMetadataTemplate`.

Reply:
Weird. Looks Ok for me...
![delete-btn](https://user-images.githubusercontent.com/1250790/53898009-835d2300-3ff4-11e9-9192-6f88d91a0125.png)

Google chrome - Version 72.0.3626.121 (Official Build) (64-bit)

Reply:
It does look like your screenshot in storybook 😕

Reply:
![delete-with-margin](https://user-images.githubusercontent.com/1250790/53899503-93c2cd00-3ff7-11e9-99eb-ce1e4f7d9b07.png)


Fixed it with margin. Looks good in Belphe as well.

Reply:
I think the reason it looked OK in your 1st screenshot is there is only an attribute name header, but no attribute description subheading.

Adding the margin appears to fix it, though.

Nice find! 👍